docs/reference/contrib/settings.rst

```diff
@@ -219,3 +222,47 @@ Or, alternately, using the ``set`` tag:
 .. code-block:: html+jinja
 
     {% set social_settings=settings("app_label.SocialMediaSettings") %}
+
+
+Utilising ``select_related`` to improve efficiency
+--------------------------------------------------
+
+For models with foreign key relationships to other objects (e.g. pages),
+which are very often needed to output values in templates, you can set
+the ``select_related`` attribute on your model class to have Wagtail
+fetch the related data for those fields when looking up setting for a site.
+
+Doing this results in a more complex initial query, but means that further
```

Comment:
This reads overly complex. How about something like:
"By doing so, the initial query will be more complex, but it will reduce additional database queries when accessing the related field values." ?

And maybe link to Django's `select_related` docs for further reading?